cloudformation/netagent2/README.md

```diff
@@ -0,0 +1,58 @@
+# Deploy AccessTier using AWS CloudFormation
+
+### Prerequisites
+
+* `Amazon Web Services (AWS)` account
+* Permissions to launch `EC2` instances, manage `AutoScaling` and set `Security Groups`.
+
+### Step 1 - Ensure AWS Networking
+
+Ensure that your VPC has an Internet Gateway attached and a Public Subnet where you can deploy the AccessTier.
+
+
+### Step 2 - Create the AccessTier Stack using CloudFormation in the AWS Console
+
+Navigate in the AWS Console to CloudFormation > Create stack.
+
+Upload the file `banyan-access-tier-v2-ubuntu.json` or `banyan-access-tier-v2-amazon-linux.json`.
+
+
+### Step 3 - Enter the Parameters required and deploy AccessTier
+
+Provide a `Stack name` (which will also be the access tier name), and specify the `VPC` into which the AccessTier should be deployed as well other relevant details.
```

Comment:
Perhaps add some indication that this access tier _must_ be defined within the command center before running the CF template.

Reply:
updated